src/env.h

```diff
@@ -7,6 +7,7 @@
 #include "debug-agent.h"
 #if HAVE_INSPECTOR
 #include "inspector_agent.h"
+#include <vector>
```

Comment:
Yes, I had to make sure they do not leak outside of inspector_agent translation unit. This issue might've been fixed since.